3p/github.js

```diff
@@ -45,8 +45,13 @@ export function github(global, data) {
     data.element);
 
   const gistid = data.gistid;
+  var gistUrl = 'https://gist.github.com/' + gistid + '.js';
 
-  getGistJs(global, 'https://gist.github.com/' + gistid + '.js', function() {
+  if(data.file) {
+    gistUrl += '?file=' + data.file;
```

Comment:
`encodeUriComponent` the `data.file`. Also do it for `gistid` (Just in case patterns change later to include url unsafe chars)